hub3/ead/mets.go

```diff
@@ -90,6 +90,17 @@ func updateFileInfo(files map[string]*eadpb.File, fg []*CfileGrp, fa *eadpb.Find
 	for _, grp := range fg {
 		if strings.EqualFold(grp.AttrUSE, "default") {
 			defaultGrp = grp
+		} else if strings.EqualFold(grp.AttrUSE, "display") {
+			for _, metsFile := range grp.Cfile {
+				id := strings.TrimSuffix(strings.TrimPrefix(metsFile.AttrID, "ID"), "IIP")
+				file, ok := files[id]
+				if !ok {
+					return fmt.Errorf("id should always be in webresource map: %s", id)
```

Comment:
Can you change this format string to a constant. 

The two code-smells reported by sonar should be fixed before I can merge this.